### DISCUSSION TITLE
Chinese investors lead foreigners with $6.1 billion in purchased US real estate last year

### DISCUSSION THREAD

**Discussion by suncunhu** [WeNeedHome/SummaryOfLoanSuspension#1016] [Ideas]
[Chinese investors lead foreigners with $6.1 billion in purchased US real estate last year](https://finance.yahoo.com/news/chinese-investors-lead-foreigners-6-221101317.html)

张三买了美国大豪斯，李四买了中国烂尾楼，他们都有美好的未来。 :)

PS. 张三买大豪斯的钱是逼李四做三十年房奴换来的。

**Comment by zxcs001** [comment]
共产党红色血脉把老百姓当韭菜往四里面割，让全民背上几十年负债，然后把资产全部放到瑞士银行呗。

中国人（99%， 包括什么体制内的，本质上都是些打手给红色血脉干活的狗罢了）都是些牛马，牲畜，作为人唯一的价值就是提供生产力，为了红色贵族积累在国际上能够交换的财富罢了。你拿的那些红票票都是些代金券而已，为什么正儿八经的红色血脉都不屑这些东西？因为他们知道，要真是印钞票的话，他们想印多少就印多少，这玩意可以分分钟变的一分钱都不值。所以正儿八经的常委之类的，都是直接持有资产和美金。这些玩意才是实打实有价值的东西。

所以看清楚这一点的人（不是有钱人就是高知、体制内中高层），早就有打算了，不过大多数人还是看不清形势。

你没看这些天，Youtube上那些国人失业的，贷款还不上家破人亡的视频。大家基本上都是在看笑话的心态来看的。

看中国人被铁拳砸的生不如死，成了不少海外网民每日娱乐的一部分了……
